frontend/src/concepts/pipelines/content/tables/pipelineVersion/PipelineVersionTable.tsx

```diff
@@ -67,6 +67,9 @@ const PipelineVersionTable: React.FC<PipelineVersionTableProps> = ({
         columns={pipelineVersionColumns}
         rowRenderer={(version) => (
           <PipelineVersionTableRow
+            // TODO: could version id be the same as the pipeline id?
```

Comment:
no in v2 pipeline id and version id are different. a version object will have params for `pipeline_version_id` and pipeline_id

Reply:
Great, removing the comment.